bedrock/firefox/templates/firefox/sync.html

```diff
@@ -212,9 +214,12 @@
   <div class="inner-wrapper">
     <div class="container">
       <header>
-        <h3>{{ _('Have an <b>Android smartphone</b> or <b>tablet</b>?') }}</h3>
-        <a href="{{ settings.GOOGLE_PLAY_FIREFOX_LINK }}" class="sync-button green" data-interaction="button click" data-element-action="Sync CTA" id="cta-android-footer">
-          {{ _('DOWNLOAD FIREFOX FOR ANDROID') }}
+        <h3>{{ _('Get Firefox everywhere') }}</h3>
```

Comment:
Actually, we don't even need that, because [base-resp already includes firefox/new](https://github.com/jpetto/bedrock/blob/bug-1238061-add-ios-promos-to-sync-page/bedrock/firefox/templates/firefox/base-resp.html#L5)

In fact the title is already translated at the bottom
https://www-demo5.allizom.org/it/firefox/sync/